go-controller/pkg/util/libovsdb.go

```diff
@@ -20,6 +20,17 @@ import (
 	"k8s.io/klog/v2"
 )
 
+var (
+	nbDBTableMonitors = []model.Model{
```

Comment:
i think its safe to monitor the entire nbdb, and just restrict the sbdb? I guess this may change once we pull in update2/3

Reply:
I don't doubt it's "safe", but why would we do it when we don't need them?

Reply:
My recollection on using monitorAll vs monitoring specific tables is as follows.
MonitorAll makes the life of ovsdb simpler, because it needs not to figure out what clients get what updates.
On the other hand monitorAll -- while making cpu on ovsdb server lighter -- can cause a lot of 'chatter' and the clients getting huge amounts of unwanted data. So, it is a trade-off. For OSP, we never use monitorAll. But it seems that in OCP monitor all is actually preferred because most clients need most of the updates anyways?!?

Reply:
Yes, that's correct. With MonitorAll we are trading off CPU consumption versus network traffic. I think it's fine to restrict the SBDB tables (probably) since clearly the master doesn't care about the biggest component of SBDB: Logical Flows.

On the Northbound since, we're going to want everything I think, because the master is really the only thing changing the NBDB. But that said, since the active master is really the only NB client, maybe it's OK to restrict there too.

I think we should be data driven on this; does this have a CPU impact on the NB or SB at high scale?